### PR TITLE
chore: add additional cft test cases and fix the coding style for cft.

### PR DESCRIPTION
**Pull request by CienetStingLin**
1. Add additional cft test cases to check the eventTrigger
2. Fix the coding style for cft
3. Update README describing /build files
4. Fix the year information of the license

### CODE REVIEW

File: build/int.cloudbuild.yaml [hunk 1]
```diff
@@ -0,0 +1,52 @@
+# Copyright 2023 Google LLC
```
Comment (ivanmkc) [inline build/int.cloudbuild.yaml:1]: Please update the root level README to describe what these files are for.

**Comment by bharathkkb** [comment]
@balajismaniam this is a KI with event arc. In the test framework, you can add a retry config
https://github.com/terraform-google-modules/terraform-google-bigquery/blob/7e2801bdb2272b611ddb28bcb6730ab3c5f73a24/test/integration/data_warehouse/data_warehouse_test.go#L29

**Review by xsxm — APPROVED** [review]
LGTM

**Comment by balajismaniam** [comment]
Thanks for the pointer @bharathkkb. Wanted to check whether increasing the wait time or adding retries is the better option. Sounds like retries are the way to go. Will add them and hopefully we are good.

**Pull request merged**